GeoQuizChallenge/app/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,15 @@
 <resources>
     <string name="app_name">GeoQuiz</string>
+    <string name="question_australia">Canberra is the capital of Australia.</string>
+    <string name="question_oceans">The Pacific Ocean is larger than the Atlantic Ocean.</string>
+    <string name="question_mideast">The Suez Canal connects the Read Sea and the Indian Ocean</string>
+    <string name="question_africa">The source of the Nile River is in Egypt</string>
+    <string name="question_americas">The Amazon River is the longest river in the Americas.</string>
+    <string name="question_asia">Lake Baikal is the world\'s oldest and deepest freshwater lake.</string>
     <string name="question_button_true">true</string>
-    <string name="question_text">Canberra is the capital of Australia.</string>
     <string name="question_button_false">false</string>
+    <string name="prev_button">Prev</string>
```

Comment:
as a general comment we tend if the string is a word we ussaly put the same word as the name of the string also i think the style guide recommends you to name the strings with the same text interpolated with `_ ` so `question_africa` should be `the_source_of_the_nile_river_is_in_egypt`